src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -146,6 +146,7 @@
             <argument type="service" id="api_platform.subresource_data_provider" />
             <argument type="service" id="api_platform.serializer.context_builder" />
             <argument type="service" id="api_platform.identifier.converter" />
+            <argument type="service" id="event_dispatcher" />
```

Comment:
`symfony/event-dispatcher` must then be added to the `require` section of `composer.json` to avoid a transient dependency.